consensus/conf.bzl

```diff
@@ -1,4 +1,5 @@
 CONSENSUS_MAINNET_VARIABLES = [
+    "DB_PATH='\"ciri/ciri.db\"'",
```

Comment:
That's really great, now that you're at it, can you make it `ciri/ciri-mainnet.db` ?